src/components/Badge/sgds-badge.ts

```diff
@@ -38,10 +46,17 @@ export class SgdsBadge extends SgdsElement {
   /** Manually set the dismissible state of the button to `false` */
   @property({ type: Boolean, reflect: true }) dismissible = false;
 
+  /** Manually enable full width */
```

Comment:
describe what will happen when full width? like badge will take the width of the parent container etc..